multi-node/aws/cmd/kube-aws/command_up.go

```diff
@@ -26,7 +26,7 @@ var (
 func init() {
 	cmdRoot.AddCommand(cmdUp)
 	cmdUp.Flags().BoolVar(&upOpts.export, "export", false, "Don't create cluster, instead export cloudformation stack file")
-	//	cmdUp.Flags().BoolVar(&upOpts.update, "update", false, "update existing cluster with new cloudformation stack")
+	cmdUp.Flags().BoolVar(&upOpts.update, "update", false, "update existing cluster with new cloudformation stack")
```

Comment:
The other two flags start with capital letters.  I believe the rest of the flags in the project are all lowercase.  Looks like none use punctuation.

We should decide on what flags should look like and make them consistent (ideally across all our projects).  I'd vote for capitalizing without punctuation by way of cargo-culting cargo/rustc.  Thoughts @robszumski ?